store/mongo/datastore_mongo.go

```diff
@@ -740,6 +740,45 @@ func (db *DataStoreMongo) GetFiltersAttributes(ctx context.Context) ([]model.Fil
 	return attributes, nil
 }
 
+func (db *DataStoreMongo) DeleteGroup(
+	ctx context.Context,
+	group model.GroupName,
+) (chan model.DeviceID, error) {
+	deviceIDs := make(chan model.DeviceID)
+
+	database := db.client.Database(mstore.DbFromContext(ctx, DbName))
+	collDevs := database.Collection(DbDevicesColl)
+
+	filter := bson.M{DbDevAttributesGroupValue: group}
+	findOptions := &mopts.FindOptions{
+		Projection: bson.M{DbDevId: 1},
+	}
+	cursor, err := collDevs.Find(ctx, filter, findOptions)
+	if err != nil {
+		return nil, err
+	}
+	go func() {
```

Comment:
Should we check if reporting exist before forking this go routine?

Reply:
I don't think so, and the datastore doesn't know anything about reporting.